### PR TITLE
Improve comments output

### PR DESCRIPTION
**Pull request by mattwr18**
before, collpased:
Screenshot from 2019-09-23 09-03-00

uncollapsed:
Screenshot from 2019-09-23 09-03-10

after, collapsed:
Screenshot from 2019-09-23 09-03-31

uncollapsed:
Screenshot from 2019-09-23 09-03-39

### CODE REVIEW

File: webapp/components/Comment.vue [hunk 1]
```diff
@@ -38,14 +38,14 @@
...
-          v-if="comment.content.length < 400"
+          v-if="$filters.removeHtml(comment.content).length < 180"
           :content="comment.content"
           class="padding-left"
         />
         <div v-else class="show-more-or-less-div">
           <content-viewer
             v-if="isCollapsed"
-            :content="$filters.truncate(comment.content, 400)"
+            :content="$filters.truncate(comment.content, 180)"
```
Comment (roschaefer) [inline webapp/components/Comment.vue]: Please refactor the 180 to be not such a magic number. You could put it into `constants/comments.js`.

File: webapp/package.json [hunk 2]
```diff
@@ -77,6 +77,7 @@
     "tippy.js": "^4.3.5",
     "tiptap": "~1.25.0",
     "tiptap-extensions": "~1.27.0",
+    "trunc-html": "^1.1.2",
```
Comment (roschaefer) [inline webapp/package.json]: Very good. You chose the one which has more Github stars than the other two options.

Comment (mattwr18) [reply webapp/package.json]: hehe, I chose the one we use in the `excerptMiddleware`, but whoever chose it must have looked at the GitHub stars :smile_cat:

File: webapp/components/Comment.vue (hunk 1 above)
Comment (Tirokk) [reply webapp/components/Comment.vue]: I agree with @roschaefer . 😍

File: webapp/plugins/vue-filters.js [hunk 3]
```diff
@@ -45,9 +46,9 @@ export default ({ app = {} }) => {
       if (length <= 0) {
         return value
       }
-      let output = value.substring(0, length)
+      let output = trunc(value, length).html
       if (output.length < value.length) {
-        output += '…'
+        output += ' …'
```
Comment (Tirokk) [inline webapp/plugins/vue-filters.js]: I really love this @mattwr18 ! 💫😍
The ellipse the most and for sure the space as well. 💥

**Pull request merged**